packages/block-editor/src/components/block-tools/zoom-out-mode-inserters.js

```diff
@@ -62,7 +68,19 @@ function ZoomOutModeInserters() {
 	const index = blockOrder.findIndex(
 		( clientId ) => selectedBlockClientId === clientId
 	);
-	const nextClientId = blockOrder[ index + 1 ];
+
+	const insertionIndex = index + 1;
+
+	const nextClientId = blockOrder[ insertionIndex ];
+
+	// if the block insertion point is visible, and the insertion
+	// indicies match then we don't need to render the inserter.
```

Comment:
```suggestion
	// indices match then we don't need to render the inserter.
```
Super nit

Reply:
Missed this one. Apologies

Reply:
@MaggieCabrera Please can you approve https://github.com/WordPress/gutenberg/pull/68051